metricflow/dataflow/nodes/alias_specs.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from abc import ABC
+from dataclasses import dataclass
+from typing import Sequence, Tuple
+
+from metricflow_semantics.dag.id_prefix import IdPrefix, StaticIdPrefix
+from metricflow_semantics.dag.mf_dag import DisplayedProperty
+from metricflow_semantics.specs.instance_spec import InstanceSpec
+from metricflow_semantics.visitor import VisitorOutputT
+
+from metricflow.dataflow.dataflow_plan import DataflowPlanNode
+from metricflow.dataflow.dataflow_plan_visitor import DataflowPlanNodeVisitor
+
+
+@dataclass(frozen=True, eq=False)
+class AliasSpecsNode(DataflowPlanNode, ABC):
+    """Change the columns matching the key specs to match the value specs."""
+
+    change_specs: Sequence[Tuple[InstanceSpec, InstanceSpec]]
```

Comment:
Since this class is frozen, it's preferable to keep the field types immutable as well. i.e.

`change_specs: Tuple[...]`

Also helpful to have a dataclass for `Tuple[InstanceSpec, InstanceSpec]` so that you don't have to remember what index is what.